packages/analytics/package.json

```
@@ -6,19 +6,27 @@
   "type": "module",
   "license": "GPL",
   "scripts": {
-    "start": "node src/index.js",
+    "start": "node --loader ts-node/esm src/index.ts",
     "build": "tsc",
-    "dev": "nodemon src/index.js"
+    "watch": "tsc -w",
+    "dev": "nodemon --loader ts-node/esm src/index.ts --trace-warnings"
```

```suggestion
    "dev": "nodemon"
```
We defined nodemon config so stick with that.